src/shared/types/abTests/epic.ts

```diff
@@ -126,7 +126,6 @@ export const epicTestFromToolSchema = testSchema.extend({
     articlesViewedSettings: articlesViewedSettingsSchema.optional(),
     priority: z.number(),
     variants: variantSchema.array(),
-    isBanditTest: z.boolean().optional(),
```

Comment:
there are currently no tests using this field, so I'm replacing it with the `methodologies` field which better models the capabilities